magenta/models/music_vae/js/demo/index.ts

```diff
@@ -180,10 +183,56 @@ async function runMel(){
   console.log(dl.memory());
 }
 
+function concatNoteSequences(
```

Comment:
We should just port sequences_lib.concatenate_sequences and put it in core.

Reply:
Agreed, but I don't want to block this since that is a more complicated version of the function and will need tests. Added a TODO for now.